crates/contract/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
```

Comment:
Even though unrelated to this PR, This makes sense to me to commit as a package.